### PR TITLE
Customisation of django's admin

### PR DESCRIPTION
**Pull request by szok**
Added custom:
- lists,
- forms,
- templates

Connected to #1433

### CODE REVIEW

File: src/ralph/settings/base.py [hunk 1]
```diff
@@ -49,9 +51,9 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [
-            os.path.join(BASE_DIR, 'templates'),
-        ],
+        # 'DIRS': [
+        #     os.path.join(BASE_DIR, 'templates'),
+        # ],
```
Comment (ghost) [inline src/ralph/settings/base.py]: Is this comment a mistake?

Comment (szok) [inline src/ralph/settings/base.py]: Done.

File: src/ralph/urls/base.py [hunk 2]
```diff
@@ -6,9 +6,9 @@
 from __future__ import unicode_literals
 
 from django.conf.urls import include, url
-from django.contrib import admin
 
+from ralph.admin import ralph_site as admin
 
 urlpatterns = [
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', include(admin.urls)),
```
Comment (mkurek) [inline src/ralph/urls/base.py]: why not just `/`?

File: src/ralph/lib/foundation/templatetags/foundation_form.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+from django import template
+
+
+register = template.Library()
+
+
+@register.inclusion_tag('foundation_form/form_label.html')
+def label(field):
+    """Render foundation field."""
```
Comment (ghost) [inline src/ralph/lib/foundation/templatetags/foundation_form.py]: perhaps copy-paste docstring, update or delete?

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**